src/bvals/cc/bvals_cc_in_one.cpp

```diff
@@ -292,6 +278,18 @@ void ResetSendBufferBoundaryInfo(MeshData<Real> *md, size_t buffers_used) {
   auto boundary_info = BufferCache_t("send_boundary_info", buffers_used);
   auto boundary_info_h = Kokkos::create_mirror_view(boundary_info);
 
+  // TODO(JMM): The current method relies on an if statement in the par_for_outer.
+  // Revisit later?
+
+  // Get coarse and fine bounds. Same for all blocks.
+  auto &rc = md->GetBlockData(0);
+  auto pmb = rc->GetBlockPointer();
```

Comment:
Potentially add scope or rename `pmb` as `pmb` is also used in the following for loop.

Reply:
This one is essentially a throw-away, and it's fine that it's shadowed in scope. We only use it to get the index shapes, etc. But I agree, it could be hidden in `{}`.